app/views/images/index.html.haml

```diff
@@ -0,0 +1,8 @@
+- if @images.any?
+  - @images.each do |image|
+    %h2= image.name
+    = image_tag image.image.url(:thumb)
+    = link_to image.name, image_path(image)
+- else
+  No images yet
+%a(href='images/new') Add an image
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))